modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
+A tool to configure and manage {gitops-title} and Argo CD resources from the command line. 
+
+Control plane ({gitops-shortname} control plane)::
+In the {gitops-shortname} context, control plane is the `openshift-gitops` namespace, where, by default, the {gitops-title} installs an instance of Argo CD. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. Examples include a control plane repository wherein you can create one or more application custom resources and express an intent to deploy and sync Kubernetes manifests from Git.
++
+Within the `openshift-gitops` namespace, Argo CD maintains a set of following Kubernetes resources, which define the continuous deployment between the source Git repository and destination clusters:
+
+* Argo CD `Application` custom resources (CRs)
+* `ConfigMap` API objects
+* `Secret` objects representing the {gitops-shortname} repository credentials and cluster credentials for deployment targets
+
+Manifest::
+In the {gitops-shortname} context, a manifest is a YAML representation of Kubernetes resources defined within a {gitops-shortname} repository, with the intent to deploy those resources to a target Kubernetes cluster. An example of a manifest would be the YAML representation of resources such as `Deployment`, `ConfigMap`, or `Secret`.
+
+Workload::
+Any process usually defined within resources such as `Deployment`, `StatefulSet`, `ReplicaSet`, `Job`, or `Pod`, running within a container. Examples include a Spring Boot application, a NodeJS Express application, or a Ruby on Rails application.
+
+Application::
+A group of Kubernetes resources as defined by a manifest. This is a Custom Resource Definition (CRD). An `Application` CRD is a resource object representing a deployed application instance in an environment.
+
+Argo CD application::
+An Argo CD application tracks the continuous deployment of individual Kubernetes resources from the {gitops-shortname} repository, where the resources are defined as manifests, to a target Kubernetes cluster.
+
+Declarative setup::
+A declarative description of the infrastructure you need in your specified environment, for system and application setup or configuration. You can specify this description in a YAML configuration file in the Git repository. The declarative setup contains an automated process to make your environment and infrastucture match the described state. Examples include defining Argo CD applications, projects and settings declaratively using YAML manifests.
+
+Live state::
+The live state of application resources on a target cluster.
+
+Target state::
+The desired state of application resources, as represented by files in a Git repository.
+
+Sync::
+The process of synchronizing the live state of an application's cluster resources with the target state defined within Git to ensure consistency, by using the Argo CD UI. Examples include syncing an application by applying changes to a cluster.
```

Comment:
Suggestion: Discard if you feel otherwise:

defined within Git to ensure consistency -> defined within the Git repository to ensure consistency?